roles/confluent.test/tasks/check_cluster_registry.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: Get Kafka Cluster Id
+  include_tasks: ../../tasks/rbac_setup.yml
+  vars:
+    copy_certs: false
+
+- name: Retrieve Cluster Registry details
+  uri:
+    url: "{{mds_http_protocol}}://{{ groups['kafka_broker'][0] }}:{{mds_port}}/security/1.0/registry/clusters"
+    method: GET
+    validate_certs: false
+    force_basic_auth: true
+    url_username: "{{mds_super_user}}"
+    url_password: "{{mds_super_user_password}}"
+    status_code: 200
+  register: cluster_name_query
+
+- set_fact:
+    cluster_name_query_json: "{{ cluster_name_query.json }}"
+
+- name: Assert Broker Cluster ID as expected
+  assert:
+    that:
+      - broker_id == "{{cluster_name_query_json | json_query('[0].clusterName')}}"
+    fail_msg: "Broker ID set to {{cluster_name_query_json | json_query('[0].clusterName')}} not {{broker_id}}"
+    quiet: true
+
+- name: Assert Kafka Connect Cluster ID as expected
+  assert:
+    that:
+      - connect_id == "{{cluster_name_query_json | json_query('[1].clusterName')}}"
+    fail_msg: "Connect Cluster ID set to {{cluster_name_query_json | json_query('[1].clusterName')}} not {{connect_id}}"
+    quiet: true
+
+- name: Assert KSQL Cluster ID as expected
+  assert:
+    that:
+      - ksql_id == "{{cluster_name_query_json | json_query('[2].clusterName')}}"
+    fail_msg: "KSQL Cluster ID set to {{cluster_name_query_json | json_query('[2].clusterName')}} not {{ksql_id}}"
+    quiet: true
+
```

Comment:
no Rest Proxy cluster registry?

Reply:
Good question, we don't mention it in the cluster registry documentation at all, and Rest Proxy itself does not support clustering, in that each instance is unique and does not/can not rely on any other instance.  https://docs.confluent.io/current/kafka-rest/production-deployment/rest-proxy/index.html#deployment

That being said, you still may want to grant each instance a unique/simple name.  Let me ask the team.

Reply:
So confirmed, we don't model anything in cluster registry for Rest Proxy as it also doesn't have dedicated RBAC requirements either.